src/client/components/chart-detail-page-layout/chart-detail-page.css

```diff
@@ -0,0 +1,20 @@
+ .chart-details{
+    display: flex;
+    flex-direction: column;
+      justify-content:space-around; 
+      height:85vh; 
+     margin-left: 200px; 
```

Comment:
Those margins are too big, and it is better to use em than px
Once you use wrapper as @gizemcandemir suggested, it would be good to remove margins from here.